internal/github/github_test.go

```diff
@@ -0,0 +1,70 @@
+package github
+
+import (
+	"context"
+	"testing"
+)
+
+func TestLanguagesStruct(t *testing.T) {
```

Comment:
This seems to be testing Go's standard library, rather than behaviour we've implemented ourselves. I'm not sure `TestLanguagesStruct` is needed